apps/www/graphql/republik-api/queries/DocumentProgressQuery.gql

```diff
@@ -0,0 +1,14 @@
+query getDocumentProgress($path: String!) {
+  document(path: $path) {
+    id
+    userProgress {
+      percentage
```

Comment:
Nit: Are we using this value anywhere? If not, maybe we don't need to fetch it.

Reply:
sooo... it doens't upsert document progress when the current node's percentage is the same as the userprogress??? might be a mistake